client/components/main/index.jsx

```diff
@@ -11,7 +11,7 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<main className={ classnames( this.props.className, 'main' ) } role="main">
+			<main className={ classnames( this.props.className, 'main' ) } role="main" style={ this.props.style }>
```

Comment:
@mtias: Yeah I put this in there because the page needs to be able to expand beyond it's defined [`max-width`](https://github.com/Automattic/wp-calypso/blob/master/client/components/main/style.scss#L3); I have a feeling this will start to happen more & more.

The style `propType` doesn't have to be the only way go though, we could use an `expanded` boolean prop and let that dictate the actual width (`&.is-expanded { max-width: 1040px; }`), etc.